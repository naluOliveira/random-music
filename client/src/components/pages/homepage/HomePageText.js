import React from 'react';

export default function HomePageText() {
  return (
    <div className='w3-col homepage homepage-container'>
      <div className='homepage-content homepage-left-content'>
        <h1 className='homepage-content-title'>
          Gerador de músicas aleatórias
        </h1>
        <div className='homepage-content-text'>
          <p>
            Aqui você pode conhecer uma música completamente aleatória todos os
            dias! Você também pode gerar uma músicas baseadas no seu gosto
            musical, usando suas playlists salvas ou músicas recém tocadas.
          </p>
        </div>
      </div>
    </div>
  );
}
