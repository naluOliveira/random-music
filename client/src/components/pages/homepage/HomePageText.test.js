import React from 'react';
import { screen, render } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import HomePageText from './HomePageText';

describe('', () => {
  const renderComponent = () => {
    return render(<HomePageText />);
  };

  test('uses jest-dom', () => {
    const { asFragment } = renderComponent();

    expect(asFragment()).toMatchSnapshot();
  });
});
