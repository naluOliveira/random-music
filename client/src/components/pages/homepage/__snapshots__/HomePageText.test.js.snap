// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`uses jest-dom 1`] = `
<DocumentFragment>
  <div
    class="w3-col homepage homepage-container"
  >
    <div
      class="homepage-content homepage-left-content"
    >
      <h1
        class="homepage-content-title"
      >
        Gerador de músicas aleatórias
      </h1>
      <div
        class="homepage-content-text"
      >
        <p>
          Aqui você pode conhecer uma música completamente aleatória todos os dias! Você também pode gerar uma músicas baseadas no seu gosto musical, usando suas playlists salvas ou músicas recém tocadas.
        </p>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
